src/anm/player.js

```diff
@@ -385,6 +386,7 @@ Player.prototype.load = function(arg1, arg2, arg3, arg4) {
                     if (player.anim === result) { // avoid race condition when there were two requests
                         // to load different animations and first one finished loading
                         // after the second one
+                        player.playedOnce = false;
```

Comment:
why not assign `playedOnce` outside the condition before even bothering with remotes?